components/dom_crawler.rst

```diff
@@ -620,6 +620,21 @@ the whole form or specific field(s)::
     $form->disableValidation();
     $form['country']->select('Invalid value');
 
+Resolving URI
+~~~~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The `UriResolver` helper was added in Symfony 5.1.
+
+The :class:`Symfony\\Component\\DomCrawler\\UriResolver` class takes an URI
+(relative, absolute, fragment, etc.) and turns it into an absolute URI against
+another given base URI.::
+
+    UriResolver::resolve('/foo', 'http://localhost/bar/foo/'); // => 'http://localhost/foo'
+    UriResolver::resolve('?a=b', 'http://localhost/bar#foo'); // => 'http://localhost/bar?a=b'
+    UriResolver::resolve('../../', 'http://localhost/'); // =>  http://localhost/'
```

Comment:
Missing single quote before output, but for readability let’s remove the „the“ and the single quotes completely